components/Image.tsx

```diff
@@ -0,0 +1,25 @@
+// Code taken from GitHub discussion thread on Next/Image
+// https://github.com/vercel/next.js/discussions/18739
+
+import styles from '/styles/Image.module.scss';
+import NextImage, { ImageProps } from 'next/image';
+
+export const Image = ({ className = '', ...props }: ImageProps) => {
+    // I've specified a default empty string value for alt text here
```

Comment:
Thank you, that's a nicer way of writing it :)

Reply:
oh cool, I think it should be fine, `...props` is just the same as the other props being passed in and we can modify them as we like as the code will run before the html/JSX is returned. If this wasn't a simple component then we'd want to use something like a `useState` or `useEffect` hook to modify state instead to prevent running the if statement multiple times.